test.yml

```diff
@@ -0,0 +1,40 @@
+config:
+  target: 'http://localhost:3001/videos'
+  http:
+    pool: 10
+  phases:
+    - duration: 60
+      arrivalRate: 10
+      rampTo: 50
+    - duration: 60
+      arrivalRate: 50
+  defaults:
+    headers:
+      # x-my-service-auth: '987401838271002188298567'
+  payload:
+    path: "id.csv"
+    fields:
+      - "id"
+    skipheader: true
+scenarios:
+  - name: "GET requests"
+    weight: 19
+    flow:
```

Comment:
Nice weighting of scenarios Aysun! You could go even lower than 5% if you wanted to.

Do you think waiting 5 seconds after get request is realistic? Or do you feel we should be making it longer/shorter?